meteor/lib/api/userActions.ts

```diff
@@ -160,6 +161,7 @@ export interface NewUserActionAPI extends MethodContext {
 	restartCore(userEvent: string, token: string): Promise<ClientAPI.ClientResponse<string>>
 	guiFocused(userEvent: string, viewInfo?: any[]): Promise<ClientAPI.ClientResponse<void>>
 	guiBlurred(userEvent: string, viewInfo?: any[]): Promise<ClientAPI.ClientResponse<void>>
+	getTranslationBundle(bundleId: TranslationsBundleId): Promise<ClientAPI.ClientResponse<TranslationsBundle>>
```

Comment:
I don't think it's a good idea to use the UserActions API for fetching the translations, as that is tied to user-facing things (since it logs an entry to the userActions-log). For this I would instead just expose a plain Meteor method. It's not as type-safe, but it is what we have.

Reply:
I agree, the purpose of the UserActions API is to provide a record of what the user did, and this does not want to be logged like that